### ISSUE TITLE
Setup breaks with server version 5.1.4+

### ISSUE THREAD

**Issue by Fuco1** [lookyman/nette-oauth2-server-doctrine#13]
In versions 5.1.4+ and 6.0+ there were some changes to the setup: https://oauth2.thephpleague.com/v5-security-improvements/

specifically one has now to call 

```php
$server->setEncryptionKey('lxZFUEsBCJ2Yb14IF2ygAHI5N4+ZAUXXaSeeJm6+twsUmIen');
```

(where the string is some random data)

before adding grants or doing anything with the server.

On the 5 series this generates a deprecation warning only (which we treat as errors in our codebase).  I can pin for the lower version for now but it would be nice to have this fixed :)

**Comment by lookyman** [comment]
Can you please test it with https://github.com/lookyman/nette-oauth2-server-doctrine/pull/14?

I have a huge refactoring in the works, which will be for 6.0+ only, but it needs more work. So this should help in the meantime.

**Comment by Fuco1** [comment]
The fix in #14 works, it installs 5.1.3 for me.

**Comment by lookyman** [comment]
Thanks! https://github.com/lookyman/nette-oauth2-server-doctrine/releases/tag/v2.0.2
